meta-lmp-base/recipes-containers/docker-app/docker-app_git.bb

```diff
@@ -5,26 +5,25 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=4859e97a9c7780e77972d989f0823f28"
 
 GO_IMPORT = "github.com/docker/app"
-SRC_URI = "git://${GO_IMPORT} \
-	file://0001-packing-use-hub.foundries.io-cnab-app-base-for-cnab-.patch;patchdir=src/${GO_IMPORT} \
-	file://0001-Warn-on-unused-parameters-rather-than-exit.patch;patchdir=src/${GO_IMPORT} \
-"
-SRCREV = "7eea32b735b1830ca7de723a1ecb923eae65a2f7"
+SRC_URI = "git://${GO_IMPORT}"
+SRCREV = "9f9c6680050949f4298a035314fec460d647f709"
 
 UPSTREAM_CHECK_COMMITS = "1"
-PV = "v0.8.0"
+PV = "v0.6.0"
 
 inherit go goarch
 
 do_compile() {
 	cd ${S}/src/${GO_IMPORT}
 	BUILD_TAG=`git describe --always --abbrev=10`
 	BUILD_COMMIT=`git rev-parse --short HEAD`
-	DOCKER_APP_LDFLAGS="-X ${GO_IMPORT}/internal.GitCommit=${BUILD_COMMIT} \
+	DOCKER_APP_LDFLAGS="-s -w -X ${GO_IMPORT}/internal.GitCommit=${BUILD_COMMIT} \
```

Comment:
Why the change here to strip during build? OE already calls strip, which is why the insane skip wasn't required before.

Reply:
I was getting an stripping error when I built this as a test

Reply:
@ricardosalveti testing the new version of this PR locally